json-p/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.javaee8</groupId>
+        <artifactId>samples-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    
+    <artifactId>json-p</artifactId>
+    <packaging>pom</packaging>
+    <name>Java EE 8 Samples: JSON-P</name>
+
+    <modules>
+        <module>patch</module>
+        <module>pointer</module>
+        <module>merge</module>
+    </modules>
+
+    <properties>
+        <json.version>1.1</json.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.javaee8</groupId>
+            <artifactId>test-utils</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.json</groupId>
```

Comment:
This one should have been defined at the top level and then replaced by the Java EE 8 dependency when that became available (which just happened ;))